
=================================== ERRORS ====================================
_________________________ ERROR collecting test_1.py __________________________
import file mismatch:
imported module 'test_1' has this __file__ attribute:
  C:\Users\Оля\OneDrive\Рабочий стол\Учеба\Python\Seminars\Seminar14\test_1.py
which is not the same as the test file we want to collect:
  C:\Users\Оля\OneDrive\Рабочий стол\Учеба\Python\test_1.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules
=========================== short test summary info ===========================
ERROR test_1.py
!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
1 error in 0.13s
